query_optimizer/QueryHandle.hpp

```diff
@@ -81,6 +82,20 @@ class QueryHandle {
   }
 
   /**
+    * @return The catalog database cache in the protobuf format.
+    */
```

Comment:
We could, but since `serialization::CatalogDatabase` is defined without macros, I think we don't need for now.

We could always add within the macro scope later.